action/src/checkoutPrivateRepository.ts

```diff
@@ -0,0 +1,25 @@
+import { ErrorStrings, GithubEventNames } from './stringEnums';
+import { context } from '@actions/github';
+import { exec } from '@actions/exec';
+
+/**
+ * Checkout the appropriate ref for the users changes using gh cli.
+ * @returns {Promise<void>}
+ */
+export async function checkoutPrivateRepository(): Promise<void> {
```

Comment:
yeah nit above. instead of exporting whole module/namespace, just the function.

Reply:
tsconfig compilerOptions

Reply:
ditto